src/test/java/de/medizininformatikinitiative/torch/config/TestConfig.java

```diff
@@ -133,6 +149,7 @@ PatientBatchToCoreBundleWriter patientBatchToCoreBundleWriter(CompartmentManager
 
     @Bean
     @Qualifier("fhirClient")
+    @DependsOn("containerManager")
```

Comment:
what is the @DependsOn doing? Spring has to supply the ContainerManager as parameter. So I would assume that the FHIR client automatically depends on it.